### DISCUSSION TITLE
Collaboration with Murmurations

### DISCUSSION THREAD

**Discussion by olisb** [mastodon/fediverse_auxiliary_service_provider_specifications#22] [Provider Ideas]
Hey, I love what you are doing here - it's highly related to what we have been building at [Murmurations](https://murmurations.network/), which provides a way to index and search (what we call "profiles") anything which can be described in JSON. We have ideas for an updated version of the Murmurations spec (from v2 to v3) to include JSON-LD which would make Murmurations compatible with, and able to provide search for the Fediverse. Perhaps we can chat? Please let us know / get in touch via the contact form on https://murmurations.network/
In collaboration!
Oli

**Comment by oneiros** [comment]
Hi Oli, and thank you for your kind words.

I must admit that looking at the murmurations website it is not immediately clear to me, what the benefits for our purposes would be. To be quite honest I am not even sure I understand what your project is about in the first place.

But I would be happy to hear from you and learn what you think Murmurations could provide in the context of our fediscovery project.

**Reply by olisb** [reply]
Hi, thanks for coming back to me. Murmurations is a protocol which provides a way to register JSON data with a network of connected indexes, thereby increasing opportunities for search and discovery of distributed data. It may have been a bit premature of me to reach out as we are only just starting our upgraded from v2 to v3 of the protocol which will make Murmurations compatible with JSON-LD, thereby providing a way to index Fediverse data. As such, I was thinking it could then fit into your definition of a Fediverse Auxiliary Service Providers (FASP) by providing a means to search and discover Fediverse content. 
As mentioned, we think your project sounds great and will hopefully improve search capabilities across the Fediverse ...  if there is any way Murmurations could support what you are doing we would be keen to explore how we might be able to help.
